.env.sample

```diff
@@ -4,10 +4,12 @@
 #
 # * Each host name must be unique on the network (for Windows systems you
 #   might need to remove "-" characters)
-#
-# NOTE: These variables must be consistent with config/SmrMySqlSecrets.inc
 
 MYSQL_ROOT_PASSWORD=smr
+MYSQL_USER=smr
 MYSQL_PASSWORD=smr
 MYSQL_HOST=smr-mysql
+MYSQL_DATABASE=smr_live
+MYSQL_PORT=3306
```

Comment:
It looks like `MYSQL_PORT` has been removed from this PR? If so, we should probably remove it here as well.